src/osd/PrimaryLogPG.cc

```diff
@@ -9803,9 +9803,10 @@ void PrimaryLogPG::sub_op_remove(OpRequestRef op)
 eversion_t PrimaryLogPG::pick_newest_available(const hobject_t& oid)
 {
   eversion_t v;
-
-  assert(pg_log.get_missing().is_missing(oid));
-  v = pg_log.get_missing().get_items().find(oid)->second.have;
+  pg_missing_item pmi;
+  bool is_missing = pg_log.get_missing().is_missing(oid, &pmi);
+  assert(is_missing);
```

Comment:
i think we are moving from `assert()` to `ceph_assert()`. see the thread of https://www.spinics.net/lists/ceph-devel/msg31049.html